cdn/dev/js/kmwlive.js

```diff
@@ -122,7 +122,7 @@ $(document).ready(function() {
 
     // We can't proceed any further if KMW hasn't loaded yet.
     // No point handling resizes until that's occurred.
-    if(!getKeymanWeb()) {
+    if(!getKeymanWeb() || !getKeymanWeb().osk) {
```

Comment:
There's definitely not much that can be done if KeymanWeb itself hasn't loaded.

If we wanted to be a bit fancier about the case where the `keyman` global exists but the OSK hasn't been initialized, we could make this function async instead, doing something like...

```
if(!getKeymanWeb()) {
  return;
} else if(!getKeymanWeb().osk) {
  await keyman.init();
}
```

Though that might run into a noted issue with how KeymanWeb's `init` function gets called multiple times.

Alternatively... I see there is an `afterInit` function defined within this source file.  It doesn't yet call `updateSize()`, but I see no reason it couldn't if this case is something we strongly wish to handle.

Reply:
It's probably less black-box-breaking to verify `keyman.initialized == 2`? However, what we have should at least be working for now.

Reply:
`keyman.initialized == 1` would actually be sufficient.  Also, I don't think we ever reach `2` when embedding KMW in the apps.

Reply:
`keyman.initialized != 0`?